main.py

```diff
@@ -0,0 +1,31 @@
+
+
+from pptx import Presentation
+import asyncio
+#import pptx
```

Comment:
Dont leave commented out code inside your codebase! This leads to a messy codebase